.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04]
-        pyodide-version: [0.22.1, 0.23.4, 0.24.0]
+        pyodide-version: [0.24.1, 0.25.0]
```

Comment:
Does this change set rely on 0.24? Looks to me like `loadDynlibsFromPackage` was added in 0.22 and hasn't been changed since. Also, how many Pyodides does micropip target to support? Rolling support for last two versions? and same for Pyodide supporting old micropip versions?

Reply:
It relies on the list methods of PyProxy, which was added in 0.24 (https://github.com/pyodide/pyodide/commit/1aea59f55a1bfa0a790bebc99480e006553fe5df).

> Also, how many Pyodides does micropip target to support? Rolling support for last two versions? and same for Pyodide supporting old micropip versions?

My opinion is to ensure backwards compatibility whenever possible, but make breaking changes when necessary. Maybe we can make a compatibility table for micropip, just as how we do in pytest-pyodide.

Reply:
Is there a reason that you're dropping 0.22 and 0.23?

Reply:
Oops should have read more carefully.